nixos/modules/rename.nix

```diff
@@ -63,7 +63,7 @@ with lib;
     (mkRenamedOptionModule [ "services" "tarsnap" "config" ] [ "services" "tarsnap" "archives" ])
 
     # ibus
-    (mkRenamedOptionModule [ "programs" "ibus" "plugins" ] [ "i18n" "inputMethod" "ibus" "engines" ])
+    (mkRenamedOptionModule [ "programs" "ibus" "plugins" ] [ "i18n" "input-method" "ibus" "engines" ])
```

Comment:
Ah, I didn't notice this one; this won't work, because you didn't actually change the option name (it is independent of the folder name).